### ISSUE TITLE
TypeError invalid IN Operand doc

### ISSUE THREAD

**Issue by garenor** [aldeed/meteor-autoform#831]
I have a form that I use for uploading images. I use AF for adding some metadata. When you click the upload button I use the click event to upload the file, and before that, calling a `AutoForm.validateForm()`; Using 5.0.3 this was working fine, but with 5.1 I get the following error:

`TypeError: invalid 'in' operand doc`, on line 2542 of aldeed_simple_schema. Any ideas?

**Comment by aldeed** [comment]
What is the `type` on the form? Is this fixed by 5.1.1 just released?

**Comment by garenor** [comment]
I just installed 5.1.1, and have the same issue.

It is a typeless form, and only used for some basic validation.

Chrome is a little more verbose with the errormessages:

Uncaught TypeError: Cannot use 'in' operator to search for '$pushAll' in undefined, in aldeed_simple_schema.js line 2542.

form:

````
{{#autoForm schema="shipSchema.photos" id="newShipPhoto"}}
````

schema:

````
shipSchema.photos = new SimpleSchema ({
    owner: {
        type: String,
        max: 60,
        label: "Copyrighthouder foto",
        autoform: {
            placeholder: "Naam",
        },
    },
    photoSubject: {
        type: String,
        autoform: {
            firstOption: "Selecteer een onderwerp",
            type: "select",
            options: function () {
                return shipPhotoSubjectSelect();
            }
        },
        allowedValues: shipPhotoSubjectValues(),
        label: "Onderwerp van de foto"
    },
    misc: {
        type: String,
        max: 200,
        label: "Overige informatie over deze foto",
        autoform: {
            placeholder: "Overige informatie",
            rows: 3
        }
    }

});
````

button click event:

````
'click #btnUpload': function (event, template) {
        event.preventDefault();
        if (AutoForm.validateForm('newShipPhoto')) {
        ..
        }
````

I get the error on AutoForm.validateForm();

**Comment by aldeed** [comment]
OK, can you make a simple reproduction app and post a link to the repo? I don't see what the issue would be from the code you've posted.

**Comment by garenor** [comment]
It was easily reproduced.

https://github.com/garenor/AutoForm-Issue-831

Thanks!

**Comment by BradRyan** [comment]
I also have Autoform 5.1.1 and am seeing a similar issue when calling AutoForm.validateForm(formId) against a typless form which is validating against a schema.  It used to validate without error but now I get the same error message as @garenor 

Uncaught TypeError: Cannot use 'in' operator to search for '$pushAll' in undefined.

**Comment by aldeed** [comment]
Thanks for the good repro @garenor. Fixed in autoform 5.1.2
